packages/chaire-lib-common/src/config/defaultPreferences.config.ts

```diff
@@ -542,6 +542,9 @@ const defaultPreferences: PreferencesModel = {
             defaultColor: '#0086FF'
         },
         routing: {
+            // FIXME This value is a configuration, not a preference, it should match the projection expected by the calculator (here trRouting, whence 4326)
+            // Refers to EPSG:4326 projection, WGS84 World Geodetic System, used in GPS https://epsg.io/4326
+            defaultProjection: 4326,
```

Comment:
I'll be picky here, but should the name be more something like "serverProjection", "calculatorProjection" or something along those lines?

Reply:
It's not even a preference, it should be a constant. All Transition use geojson, which uses the WSG84 coordinates system. Only data coming from the outside can be in a different projection system, but we need to convert at import. Now, where does this constant go...